fastlane/lib/fastlane/helper/crashlytics_helper.rb

```diff
@@ -24,7 +24,7 @@ def discover_crashlytics_path(params)
           path ||= Dir["./Pods/iOS/Crashlytics/Crashlytics.framework/submit"].last
           path ||= Dir["./**/Crashlytics.framework/submit"].last
 
-          if path && path.downcase.include?("crashlytics.framework")
+          if path && (path_downcase = path.downcase).include?("pods") && path_downcase.include?("crashlytics.framework")
```

Comment:
What does the `(path_downcase = path.downcase)` do?

Reply:
As there is two conditions using `path.downcase`, the result is stored in `path_downcase` variable before evaluating the first test, to be used in this first test then be reused in the second test and avoid another call to downcase function.

Reply:
Ah, I overlooked the difference between `_` and `.`. Maybe I should get some glasses :/

Reply:
As it happened, means there's a readability issue.
What do you think about changing `path_downcase` to `downcase_path` to better notice the difference? `(downcase_path = path.downcase)`

Reply:
Very good idea. Or just do it in an extra line - I wouldn't have been to sure that this would actually work (but I also don't really know ruby, so there's that...) 🙊

Reply:
Done in multi-line